pkg/rule/criteria.go

```diff
@@ -7,17 +7,9 @@ import (
 )
 
 const (
-	comparatorEQ comparator = iota
-	comparatorNQ
-	comparatorGT
-	comparatorLT
-	comparatorIN
```

Comment:
we still use these comparator constants in our updated criteria structure

Reply:
That's fine, we can re-introduce them. We change it here to not have unused constants lying around.